Backend/src/main/java/com/harmonia/backend/domain/DirectMessage.java

```diff
@@ -44,12 +45,12 @@ public class DirectMessage {
     @Generated(GenerationTime.INSERT)
     private Date timestamp;
 
-//    @JsonProperty("authorId")
-//    @Column(name = "AuthorId")
-//    private Long authorId;
-//    @JsonProperty("recipientId")
-//    @Column(name = "RecipientId")
-//    private Long recipientId;
+    @JsonProperty("authorId")
+    @Column(name = "AuthorId")
+    private Long authorId;
+    @JsonProperty("recipientId")
+    @Column(name = "RecipientId")
```

Comment:
Recipient has to be similar to author:
```
 @ManyToOne(fetch = FetchType.LAZY, optional = false)
    @MapsId("UserId")
    @JoinColumn(name = "RecipientId", foreignKey = @ForeignKey(name = "Recipient_ofPMessage_fk", value = ConstraintMode.CONSTRAINT))
    User Recipient;

Reply:
its already there but i forgot to comment them out.
nvm i created one more query so i will push these changes with new ones